.eslintrc.js

```diff
@@ -61,6 +61,10 @@ module.exports = {
     // This rule is about explicitly using `return undefined` when a function returns any non-undefined object.
     // However, since we're using TypeScript, it will yell at us if a function is not allowed to return `undefined` in its signature, so we don't need this rule.
     'consistent-return': 'off',
+
+    // Hans loves writing helper functions at the bottom of files,
```

Comment:
If you're going to tag yourself in our source files at least use your github handle so that strangers on the internet can shame you too :)